components/display/st7789v.rst

```diff
@@ -57,29 +57,26 @@ to an ESP module.
 
     For more information about the font options see: :ref:`display-fonts`.
 
-.. warning::
 
-    When using the TTGO T-Display module, the GPIO pin numbers above *cannot be changed* as they are
-    hardwired within the module/PCB.
 
 Configuration variables:
 ************************
 
 - **model** (**Required**, string): The display model to use. One of the following options:
 
-  - ``TTGO TDisplay 135x240``
+  - ``TTGO TDisplay 135x240`` Automatically specifies required pins.
   - ``Adafruit Funhouse 240x240``
   - ``Adafruit RR 280x240`` (round-rectangular display -- some pixels are "deleted" from corners to form rounded shape)
   - ``Adafruit S2 TFT FEATHER 240X135`` (requires ``power_supply`` be specified, see below)
+  - ``LILYGO T-Embed 170X320`` requires ``power_supply`` be specified, but automatically specifies ``dc_pin``, ``cs_pin``,
+    ``backlight_pin`` and ``reset_pin``.
   - ``Custom`` (see details below)
 
 - **cs_pin** (*Optional*, :ref:`Pin Schema <config-pin_schema>`): The CS pin.
-- **dc_pin** (**Required**, :ref:`Pin Schema <config-pin_schema>`): The DC pin.
-- **reset_pin** (**Required**, :ref:`Pin Schema <config-pin_schema>`): The RESET pin.
-- **height** (*Optional*, int): When ``model`` is set to "Custom", use this to specify the ``height`` of the display
-  in pixels. This option may not be specified when the ``model`` is not set to "Custom".
-- **width** (*Optional*, int): When ``model`` is set to "Custom", use this to specify the ``width`` of the display
-  in pixels. This option may not be specified when the ``model`` is not set to "Custom".
+- **dc_pin** (**Required**, :ref:`Pin Schema <config-pin_schema>`): The DC pin - required except for some boards with presets.
+- **reset_pin** (**Required**, :ref:`Pin Schema <config-pin_schema>`): The RESET pin - also has presets on some boards.
```

Comment:
Given the presets model, I think I would change these to "Optional" and make the text "... **required** if not defined by `model`"